### DISCUSSION TITLE
Tạo một nút hay một giao thức có thể router giữa các trang với nhau

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#34] [Task Delivery]
Tôi xin nhận task này :D

**Comment by nguyentantrung2000** [comment]
Chào bạn sau khi xem xét thì task này vẫn chưa có người làm nên bạn sẽ làm task này nhé.
Thông tin của task:
Là các nút ấn để chuyển giữa các trang và các bước
Yêu cầu:
-Giao diện hợp lý , phù hợp đúng theo bố cục figma
-Đảm bảo có thể hoạt động router giữa các trang 
-Deadline 11:59PM 2/8/2021

**Reply by trungdoublelift** [reply]
OK
